examples/codecparam/README.md

```diff
@@ -8,7 +8,7 @@ go get github.com/pion/mediadevices/examples/codecparam
 
 ### Open example page
 
-[jsfiddle.net](https://jsfiddle.net/z7ms3u5r/) you should see two text-areas and a 'Start Session' button
+[jsfiddle.net](https://jsfiddle.net/gh/get/library/pure/pion/mediadevices/tree/master/examples/internal/jsfiddle/audio-and-video) you should see two text-areas and a 'Start Session' button
```

Comment:
jsfiddle supports loading only from the master branch. The link will be available after merging.

Reply:
Ah, I didn't know this.